nepomuk/src/main/webapp/WEB-INF/spring/appServlet/servlet-context.xml

```diff
@@ -12,7 +12,7 @@
 		http://www.springframework.org/schema/aop http://www.springframework.org/schema/aop/spring-aop-3.2.xsd">
 
 	<mvc:annotation-driven />
-	<context:component-scan base-package="edu.asu.diging.gilesecosystem.nepomuk.web" />
+	<context:component-scan base-package="edu.asu.diging.gilesecosystem.nepomuk.web,edu.asu.diging.gilesecosystem.kafka.util" />
```

Comment:
you want the services in the root context. only scan the controller package here.